README.md

```diff
@@ -67,6 +69,14 @@ You can find  other example scenes under [**Assets/MixedRealityToolkit.Examples/
 
 [![Example Scene](Documentation/Images/MRTK_Examples.png)](Documentation/README_HandInteractionExamples.md)
 
+# MRTK Examples Hub
+
+With MRTK Examples Hub, you can try various example scenes in MRTK.
+You can find pre-built app packages for HoloLens(x86), HoloLens 2(ARM), and Windows Mixed Reality immersive headsets(x64) under [**Release Assets**](https://github.com/microsoft/MixedRealityToolkit-Unity/releases/tag/v2.0.0) folder.
```

Comment:
Maybe this is common knowledge, but if not, it might be worth having a link to:

https://docs.microsoft.com/en-us/hololens/hololens-install-apps#use-the-windows-device-portal-to-install-apps-on-hololens

For how to install the packages in the release.

Reply:
Great idea!

Reply:
Added the link.